### PR TITLE
fix: improve some wordings and cli table

### PR DESCRIPTION
**Pull request by micbar**
## Description

Improve some wordings and cli output.

### New Cli output after the fix

```
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
|            NAME            |                 UID                  | ENABLED  |          DESCRIPTION           |           CONDITION            |         ALLOWED RESOURCE ACTIONS         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| Viewer                     | b1e2218d-eef8-4d4c-b82d-0f1a1b48f3b5 | enabled  | View and download.             | exists @Resource.File          | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                | exists @Resource.Folder        | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                | exists @Resource.File &&       | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                | exists @Resource.Folder &&     | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| ViewerListGrants           | d5041006-ebb3-4b4a-b6a4-7c180ecfb17d | disabled | View, download and show all    | exists @Resource.File          | libre.graph/driveItem/path/read          |
|                            |                                      |          | invited people.                | exists @Resource.Folder        | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                | exists @Resource.File &&       | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                | exists @Resource.Folder &&     | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| SpaceViewer                | a8d5fe5e-96e3-418d-825b-534dbdf22b99 | enabled  | View and download.             | exists @Resource.Root          | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| Editor                     | fb6c3e19-e378-47e5-b277-9732f9de6e21 | enabled  | View, download, upload, edit,  | exists @Resource.Folder        | libre.graph/driveItem/children/create    |
|                            |                                      |          | add and delete.                | exists @Resource.Folder &&     | libre.graph/driveItem/standard/delete    |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/update        |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| EditorListGrants           | e8ea8b21-abd4-45d2-b893-8d1546378e9e | disabled | View, download, upload, edit,  | exists @Resource.Folder        | libre.graph/driveItem/children/create    |
|                            |                                      |          | add, delete and show all       | exists @Resource.Folder &&     | libre.graph/driveItem/standard/delete    |
|                            |                                      |          | invited people.                | @Subject.UserType=="Federated" | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/update        |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| SpaceEditor                | 58c63c02-1d89-4572-916a-870abc5a1b7d | enabled  | View, download, upload, edit,  | exists @Resource.Root          | libre.graph/driveItem/children/create    |
|                            |                                      |          | add, delete including the      |                                | libre.graph/driveItem/standard/delete    |
|                            |                                      |          | history.                       |                                | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/versions/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/update        |
|                            |                                      |          |                                |                                | libre.graph/driveItem/versions/update    |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| SpaceEditorWithoutVersions | 3284f2d5-0070-4ad8-ac40-c247f7c1fb27 | disabled | View, download, upload, edit,  | exists @Resource.Root          | libre.graph/driveItem/children/create    |
|                            |                                      |          | add and delete.                |                                | libre.graph/driveItem/standard/delete    |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/update        |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| FileEditor                 | 2d00ce52-1fc2-4dbc-8b95-a73b73395f5a | enabled  | View, download and edit.       | exists @Resource.File          | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                | exists @Resource.File &&       | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| FileEditorListGrants       | c1235aea-d106-42db-8458-7d5610fb0a67 | disabled | View, download, edit and show  | exists @Resource.File          | libre.graph/driveItem/path/read          |
|                            |                                      |          | all invited people.            | exists @Resource.File &&       | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                | @Subject.UserType=="Federated" | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| EditorLite                 | 1c996275-f1c9-4e71-abdf-a42f6495e960 | enabled  | View, download and upload.     | exists @Resource.Folder        | libre.graph/driveItem/children/create    |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/update        |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| SpaceManager               | 312c0871-5ef7-4b3a-85b6-0e4074c64049 | enabled  | View, download, upload,        | exists @Resource.Root          | libre.graph/driveItem/permissions/create |
|                            |                                      |          | edit, add, delete and manage   |                                | libre.graph/driveItem/children/create    |
|                            |                                      |          | members.                       |                                | libre.graph/driveItem/standard/delete    |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/read          |
|                            |                                      |          |                                |                                | libre.graph/driveItem/quota/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/content/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/upload/create      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/read   |
|                            |                                      |          |                                |                                | libre.graph/driveItem/children/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/versions/read      |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/read       |
|                            |                                      |          |                                |                                | libre.graph/driveItem/path/update        |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/delete |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/delete     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/versions/update    |
|                            |                                      |          |                                |                                | libre.graph/driveItem/deleted/update     |
|                            |                                      |          |                                |                                | libre.graph/driveItem/basic/read         |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/update |
|                            |                                      |          |                                |                                | libre.graph/driveItem/permissions/deny   |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+
| SecureViewer               | aa97fe03-7980-45ac-9e50-b325749fd7e6 | disabled | View only documents, images    | exists @Resource.File          | libre.graph/driveItem/path/read          |
|                            |                                      |          | and PDFs. Watermarks will be   | exists @Resource.Folder        | libre.graph/driveItem/children/read      |
|                            |                                      |          | applied.                       |                                | libre.graph/driveItem/basic/read         |
+----------------------------+--------------------------------------+----------+--------------------------------+--------------------------------+------------------------------------------+

```

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/web/pkg/theme/theme.go [hunk 1]
```diff
@@ -65,14 +65,6 @@ var themeDefaults = KV{
 				"label":    "UnifiedRoleSecureView",
 				"iconName": "shield",
 			},
-			unifiedrole.UnifiedRoleFederatedViewerID: KV{
```
Comment (micbar) [inline services/web/pkg/theme/theme.go:68]: I found no real usage of that.

Comment (fschade) [reply services/web/pkg/theme/theme.go:68]: @JammingBen FYI

**Review by fschade — APPROVED** [review]

**Comment by mmattel** [comment]
Couldn't we add a column at the beginning with the name of the unified role like `UnifiedRoleSecureViewerID`. That would ease purpose identification a lot...

File: services/web/pkg/theme/theme.go (hunk 1 above)
Comment (JammingBen) [reply services/web/pkg/theme/theme.go:68]: Hmm are you sure? I think this is necessary for displaying the correct icons for the federated roles. Or am I missing something?

**Comment by micbar** [comment]
> UnifiedRoleSecureViewerID



> Couldn't we add a column at the beginning with the name of the unified role like `UnifiedRoleSecureViewerID`. That would ease purpose identification a lot...

I added that to the table, because i cannot afford having you complaining about me 😍

**Comment by fschade** [comment]
> > UnifiedRoleSecureViewerID
> 
> > Couldn't we add a column at the beginning with the name of the unified role like `UnifiedRoleSecureViewerID`. That would ease purpose identification a lot...
> 
> I added that to the table, because i cannot afford having you complaining about me 😍

🤣 🚀 🎸

File: services/web/pkg/theme/theme.go (hunk 1 above)
Comment (micbar) [reply services/web/pkg/theme/theme.go]: Clarified with @fschade

Comment (fschade) [reply services/web/pkg/theme/theme.go]: > Hmm are you sure? I think this is necessary for displaying the correct icons for the federated roles. Or am I missing something?

it uses the regular editor as example... the difference is the condition, the id is not used (and exposed)

**Pull request merged**